src/popup/components/ProtocolIcon.vue

```diff
@@ -17,12 +17,14 @@ import {
 import { PROTOCOLS } from '@/constants';
 import type { Protocol } from '@/types';
 import AeternityIcon from '@/icons/coin/aeternity.svg?vue-component';
+import AeternityIconSecondary from '@/icons/coin/aeternity-secondary.svg?vue-component';
```

Comment:
Now we have 2 icons. Maybe one is used for the blockchain and one for the coin? If this is true I suggest to rename the files accordingly.